client/additional-methods-setup/upe-preview-methods-selector/enable-upe-preview-task.js

```diff
@@ -20,26 +24,103 @@ const EnableUpePreviewTask = () => {
 
 	const handleContinueClick = useCallback( () => {
 		setIsUpeEnabled( true ).then( () => {
-			setCompleted( true, 'setup-complete' );
+			setCompleted( true, 'add-payment-methods' );
 		} );
 	}, [ setIsUpeEnabled, setCompleted ] );
 
 	return (
 		<WizardTaskItem
-			title={ __(
-				'Enable the new WooCommerce Payments checkout experience',
-				'woocommerce-payments'
-			) }
+			title={ interpolateComponents( {
+				mixedString: __(
+					'Enable the new WooCommerce Payments checkout experience {{earlyAccessPill /}}',
+					'woocommerce-payments'
+				),
+				components: {
+					earlyAccessPill: (
+						<Pill>
+							{ __( 'Early access', 'woocommerce-payments' ) }
+						</Pill>
+					),
+				},
+			} ) }
 			index={ 1 }
 		>
-			<CollapsibleBody>
+			<CollapsibleBody className="enable-upe-preview__body">
 				<p>
-					{ __(
-						'Get early access to additional payment methods and an improved checkout experience, ' +
-							'coming soon to WooCommerce payments.',
-						'woocommerce-payments'
-					) }
+					{ interpolateComponents( {
+						mixedString: __(
+							'Get early access to additional payment methods and an improved checkout experience, ' +
+								'coming soon to WooCommerce payments. {{learnMoreLink /}}',
+							'woocommerce-payments'
+						),
+						components: {
+							learnMoreLink: (
+								<a href="?TODO">
```

Comment:
We don't have this yet. Here's the plan if you're curious: https://github.com/Automattic/woocommerce-payments/issues/2170#issuecomment-880008439